runtime/wasm/src/module/mod.rs

```diff
@@ -447,9 +447,22 @@ where
 
     /// function ipfs.cat(link: String): Bytes
     fn ipfs_cat(&mut self, link_ptr: AscPtr<AscString>) -> Result<Option<RuntimeValue>, Trap> {
-        let bytes = self.host_exports.ipfs_cat(self.asc_get(link_ptr))?;
-        let bytes_obj: AscPtr<Uint8Array> = self.asc_new(&*bytes);
-        Ok(Some(RuntimeValue::from(bytes_obj)))
+        let link = self.asc_get(link_ptr);
+        let ipfs_res = self.host_exports.ipfs_cat(link);
+        match ipfs_res {
+            Ok(bytes) => {
+                let bytes_obj: AscPtr<Uint8Array> = self.asc_new(&*bytes);
+                Ok(Some(RuntimeValue::from(bytes_obj)))
+            }
+
+            // Return null in case of error.
+            Err(e) => {
+                info!(self.logger, "Failed ipfs.cat, returning `null`";
```

Comment:
I'd be in favor of making this a `warn!`.

Reply:
My thinking is that for many subgraphs ipfs hashes that cannot be found are just a fact of life, and not something the developer should be concerned about. It might be worth promoting to warn in case of timeouts, which are highly non-deterministic and you can't tell whether the file is accessible.

Reply:
But I'd have to look into IPFS behaviour to know if we can really differentiate between a file that's not found and a file that took too long to fetch. Lets leave this as a follow up after we observe what these logs look like for real subgraphs.